### ISSUE TITLE
Blueprint API not working in suburl

### ISSUE THREAD

**Issue by udanpe** [balderdashy/sails#2892]
sails generate controller api/User (controllers/api/UserController.js)
copy(or not copy) models/User.js to models/api/User.js
shortcuts: true but methods (create destroy find etc) not working

**Comment by randallmeeker** [comment]
http://stackoverflow.com/questions/27368946/create-subfolders-in-sails-js-api-controllers/27390287#27390287

**Comment by udanpe** [comment]
How a can show all _config configuration? i know how change blueprints.js in controller before and try it but 'model' its new for me.
add to documentation +1

**Comment by sailsbot** [comment]
Thanks for posting, @udanpe.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
